cpp/src/plasma/lib/java/org_apache_arrow_plasma_PlasmaClientJNI.h

```diff
@@ -0,0 +1,123 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+/* DO NOT EDIT THIS FILE - it is machine generated */
+#include <jni.h>
+/* Header for class org_apache_arrow_plasma_PlasmaClientJNI */
+
+#ifndef _Included_org_apache_arrow_plasma_PlasmaClientJNI
+#define _Included_org_apache_arrow_plasma_PlasmaClientJNI
+#ifdef __cplusplus
+extern "C" {
+#endif
+/*
+ * Class:     org_apache_arrow_plasma_PlasmaClientJNI
+ * Method:    connect
+ * Signature: (Ljava/lang/String;Ljava/lang/String;I)J
+ */
```

Comment:
@pcmoritz What should the comment style be in this file? For C++ files we've been doing `//` everywhere else, and using Doxygen style `///` for documentation.

However, should JNI files be handled differently?

Reply:
Oh, it is generated by javah automatically

Reply:
Ok, I see.

Reply:
Does this PR include instructions for how to generate the files?

Reply:
That is only the org_apache_arrow_plasma_PlasmaClientJNI.h will by generated by javah offline, and usually we do not need to add the  instructions for how to generate, because we have to write the code of  org_apache_arrow_plasma_PlasmaClientJNI.cc, and just check in the org_apache_arrow_plasma_PlasmaClientJNI.h as well.